Arisa broke

## The Bug
#210 broke searching issues, needs to call toEpochMillis when constructing the jql
```
12:16:15.288 | main | ERROR | Arisa | 58 | Failed to execute modules
java.lang.RuntimeException: net.rcarz.jiraclient.JiraException: Failed to search issues
    at net.rcarz.jiraclient.Issue$IssueIterator.hasNext(Issue.java:571)
    at net.rcarz.jiraclient.Issue$SearchResult.<init>(Issue.java:697)
    at net.rcarz.jiraclient.Issue.search(Issue.java:1359)
    at net.rcarz.jiraclient.JiraClient.searchIssues(JiraClient.java:367)
    at io.github.mojira.arisa.ModuleExecutor.searchIssues(ModuleExecutor.kt:116)
    at io.github.mojira.arisa.ModuleExecutor.executeModule(ModuleExecutor.kt:83)
    at io.github.mojira.arisa.ModuleExecutor.execute(ModuleExecutor.kt:41)
    at io.github.mojira.arisa.ArisaMainKt.main(ArisaMain.kt:53)
    at io.github.mojira.arisa.ArisaMainKt.main(ArisaMain.kt)
Caused by: net.rcarz.jiraclient.JiraException: Failed to search issues
    at net.rcarz.jiraclient.Issue$IssueIterator.getNextIssues(Issue.java:646)
    at net.rcarz.jiraclient.Issue$IssueIterator.getNextIssue(Issue.java:603)
    at net.rcarz.jiraclient.Issue$IssueIterator.hasNext(Issue.java:569)
     ... 8 common frames omitted
Caused by: net.rcarz.jiraclient.RestException: 400 : {"errorMessages":["Date value '2020-05-06T12:15:17.831Z' for field 'updated' is invalid. Valid formats include: 'yyyy/MM/dd HH:mm', 'yyyy-MM-dd HH:mm', 'yyyy/MM/dd', 'yyyy-MM-dd', or a period format e.g. '-5d', '4w 2d'."],"errors":{}}
    at net.rcarz.jiraclient.RestClient.request(RestClient.java:171)
    at net.rcarz.jiraclient.RestClient.get(RestClient.java:281)
    at net.rcarz.jiraclient.Issue$IssueIterator.getNextIssues(Issue.java:644)
     ... 10 common frames omitted 
```

## Example Ticket

## Expected behavior

--- a/src/main/kotlin/io/github/mojira/arisa/ModuleRegistry.kt
+++ b/src/main/kotlin/io/github/mojira/arisa/ModuleRegistry.kt
@@ -92,7 +92,7 @@ class ModuleRegistry(jiraClient: JiraClient, private val config: Config) {
         name: String,
         config: ModuleConfigSpec,
         module: Module<T>,
-        getJql: (lastRun: Instant) -> String = { "updated > $it" },
+        getJql: (lastRun: Instant) -> String = { "updated > ${it.toEpochMilli()}" },
         requestCreator: (Issue) -> T
     ) = register(name, config, module, getJql) { issue, _ -> requestCreator(issue) }
 

